src/OrbitGl/EventTrack.cpp

```diff
@@ -92,31 +92,30 @@ void EventTrack::UpdatePrimitives(uint64_t min_tick, uint64_t max_tick, PickingM
   float track_height = layout.GetEventTrackHeight();
   const bool picking = picking_mode != PickingMode::kNone;
 
-  const Color kWhite(255, 255, 255, 255);
-  const Color kGreenSelection(0, 255, 0, 255);
+  static const Color kWhite(255, 255, 255, 255);
+  static const Color kGreenSelection(0, 255, 0, 255);
```

Comment:
Is that a clang-tidy suggested fix? Performance wise I would assume that this might be detremental since the static initialization needs to be synchronized between threads.

Reply:
In practice this is probably not noticeable. I just wanted to mention it, feel free to ignore.